src/gluonts/shell/__main__.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import click
+
+FORECASTER_BY_NAME = {
+    'deepar': 'gluonts.model.deepar.DeepAREstimator',
```

Comment:
Should I add the remaining existing ones?

Reply:
Not for now. In a next step we could rework this by introducing some plugin or registry mechanism which allows third-party packages to register their estimators with the shell. I remember that something along those lines was possible with `setup.py`.